common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -66,8 +66,8 @@ class @Problem
     detail = @el.data('progress_detail')
     status = @el.data('progress_status')
 
-    # Render 'x/y point(s)' if student has attempted question
-    if status != 'none' and detail? and detail.indexOf('/') > 0
+    # Render 'x/y point(s)' if student has attempted question 
+    if status != 'none' and detail? and (jQuery.type(detail) == "string") and detail.indexOf('/') > 0
```

Comment:
Can you add a test point for when detail is not a string to display_spec.coffee? There is already a series of related test cases in 'renderProgressState'.